packages/atomic/cypress/integration/facets/category-facet/category-facet-actions.ts

```diff
@@ -33,9 +33,72 @@ export const addCategoryFacet =
   };
 
 export function selectChildValueAt(index: number) {
-  CategoryFacetSelectors.childValue().eq(index).click();
+  CategoryFacetSelectors.childValue()
+    .find('[part="value-label"]')
+    .eq(index)
+    .then(([childValue]) => {
+      const text = childValue.textContent!;
+      cy.wrap(childValue).click();
```

Comment:
Doing `childValue.click` directly causes focus tests to fail. My guess is that `childValue.click` calls the DOM's native click function, while `cy.wrap(childValue).click()` calls Selenium's.

Reply:
Selenium -> Cypress.

But yes :) !